workspaces/editors.md

```diff
@@ -178,3 +179,38 @@ participants. If they do, click **Accept** to proceed.
 
 At this point, you'll be able to share your project and work with your partner
 in real-time.
+
+## RStudio
```

Comment:
I think we should be explicit, and say we support RStudio Server. RStudio and RStudio Server both seem to be functionally equivalent, but I don't have enough experience with them to say that they're interchangeable.